cardano-cli/src/Cardano/CLI/Types/Errors/CmdError.hs

```diff
@@ -77,16 +81,20 @@ instance Error EraBasedDelegationError where
 data EraBasedRegistrationError
   = EraBasedRegistReadError !(FileError InputDecodeError)
   | EraBasedRegistWriteFileError !(FileError ())
-  | EraBasedRegistStakeCredReadError !ShelleyStakeAddressCmdError -- TODO: Conway era - don't use legacy error type
```

Comment:
This couples `EraBasedRegistrationError` to `ShelleyStakeAddressCmdError` which has many constructors not relevant to registration errors.